indico/modules/events/layout/controllers.py

```diff
@@ -54,21 +55,58 @@ def _render_menu_entries(event, show_hidden=False):
     return tpl.menu_entries(menu_entries_for_event(event, show_hidden=show_hidden))
 
 
+def _logo_data(event):
+    return {
+        'url': url_for('event_images.logo_display', event.as_legacy),
+        'file_name': event.logo_metadata['file_name'],
+        'size': event.logo_metadata['size'],
+        'content_type': event.logo_metadata['content_type']
+    }
+
+
+def _css_file_data(css_file):
+    return {
+        'file_name': css_file.filename,
+        'size': css_file.size,
+        'content_type': css_file.content_type
+    }
+
+
 class RHLayoutLogoUpload(RHConferenceModifBase):
```

Comment:
I would really add a `RHLayoutBase` which does nothing besides inheriting from `RHConferenceModifBase`, enabling the CSRF check and setting `self.event` (maybe `event_new` would be even better) in `_checkParams`. having less references to CMB is nice for the future in case we end up replacing it with something else (which is something i could very well imagine once we move events themselves to the new DB)